Feature/#2

seopbo opened the pull request:
- `RobertaProcessor` 작성 (FULL-SENTENCES)
- 프로젝트의 모든 코드에 `black`, `isort` 적용

Refs: #2

bzantium committed:
initial commit

Commit:
[mixed] refactor: Refactor lassl
- hydra에 대한 의존성 제거
- HfArgumentParser를 사용한 코드 단순화

bzantium committed:
[python] feat: Add pretrain code
- add pretraining code for roberta
- add tok_logger on src/preprocessing.py
- fix name of dataclass to plural expression

Refs:

seopbo committed:
[etc] chore: Add poetry env
- poetry로 환경관리

seopbo committed:
[mixed] refactor: Refactor roberta preprocessing
- RoBERTa FULL SENTENCES로 dataset을 준비하는 RobertaProcessor 작성
- 그 외 프로젝트의 모든 코드에 black, isort, poetry 적용

Refs: #2

seopbo committed:
[python] fix: Fix importing class, type hint
- serialize_corpora.py의 import하는 class 이름이 잘못된 것을 수정함.
- RobertaProcessor에서 type hint가 잘못된 것을 수정함.

Refs: #2

monologg commented:
이 코드 머지되면 TPU 정상 동작 체크 시작하겠습니다. 로그가 여러개 찍히는 이슈는 다른 PR로 분리하거나 TPU PR에서 해결하면 될 것 같습니다.

seopbo commented:
> 이 코드 머지되면 TPU 정상 동작 체크 시작하겠습니다. 로그가 여러개 찍히는 이슈는 다른 PR로 분리하거나 TPU PR에서 해결하면 될 것 같습니다.

이 PR이후에 아마 @bzantium 님과 같이 from_json 기능 작업을 해야 거기서 부터 시작이 맞을 거 같습니다. to: @monologg 

cc: @bzantium @iron-ij

seopbo committed:
[python] feat: Add a field for special_tokens_mask
- serialize_corpora.py 실행시 생성되는 pyarrow Dataset에
  special_tokens_mask field를 추가함.

Refs: #2
